config/config.go

```diff
@@ -20,9 +19,11 @@ type config struct {
 	}
 }
 
-var C config
-
-func ReadConfig() {
+/*
+	Extracts the config variables.
+*/
+func ReadConfig() (*config, error) {
+	var C config
```

Comment:
No need to have `C` exported, you are only using it in this function.

Also, there's no need to declare two variables (line 26 and 27), you could simple initialize it of type *config, and that's it.

```suggestion
	var c *config
```

[EXTRA] You should never ignore errors (line 31), and remember to wrap errors from external packages when also returning an error.